ts/session/fetch_latest_release/index.ts

```diff
@@ -0,0 +1,55 @@
+// every 1 minute we fetch from the fileserver to check for a new release
+// * if there is none, no request to github are made.
+// * if there is a version on the fileserver more recent than our current, we fetch github to get the UpdateInfos and trigger an update as usual (asking user via dialog)
+
+import { isEmpty, isString } from 'lodash';
+import { ipcRenderer } from 'electron';
+import { DURATION } from '../constants';
+import { getLatestReleaseFromFileServer } from '../apis/file_server_api/FileServerApi';
+
+/**
+ * Try to fetch the latest release from the fileserver every 1 minute.
+ * If we did fetch a release already in the last 30 minutes, we will skip the call.
+ */
+const fetchReleaseFromFileServerInterval = DURATION.MINUTES * 1;
```

Comment:
not 5 minutes?

Reply:
nah, better to have it happen fast and then fall back to the 30minutes

Reply:
Cool